collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/ProjectsAdbTest.kt

```diff
@@ -0,0 +1,34 @@
+package org.odk.collect.android.feature.projects
+
+import android.app.Application
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.support.CollectTestRule
+import org.odk.collect.android.support.TestRuleChain
+import org.odk.collect.android.support.pages.MainMenuPage
+
+@RunWith(AndroidJUnit4::class)
+class ProjectsAdbTest {
+
+    val rule = CollectTestRule()
+
+    @get:Rule
+    var chain: RuleChain = TestRuleChain
+        .chain()
+        .around(rule)
+
+    @Test
+    fun clearingStorage_andReturningToApp_recreatesStorageForProject() {
```

Comment:
Here you test going back to the `MainMenuActivity` so only `onResume()` will be called. Have you tested cases where the that activity is destroyed but the app is not killed entirely so that `ApplicationInitializer` won't be called?
I'm wondering if something bad wouldn't happen earlier (in onCreate(0 for example, before you call `currentProjectViewModel.refresh()` in `onResume()`).

Reply:
Ok I think @mmarciniak90 and @kkrawczyk123 will test this scenario.